src/bridge/AbsBridge.sol

```diff
@@ -67,6 +70,13 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
         _;
     }
 
+    /// @inheritdoc IBridge
+    function postUpgradeInit() external onlyDelegated onlyProxyOwner {
```

Comment:
`_activeOutbox` is not zero due to a gas optimization, we should definitely wipe it with postUpgradeInit during the upgrade

Reply:
Good point. I removed `postUpgradeInit`, however now thinking about it, we can keep it since it can do the wiping on Orbit chains. And we just don't use it as part of arb upgrade

Reply:
Agreed to not have `postUpgradeInit` at all to have consistent slot values (dirty context+activeOutbox slots) across all chains

Reply:
when the upgrade is executed wouldn't it go EOA -> L1Timelock -> UE? I think we could wipe it if we want but i don't see anything wrong with leaving it dirty either

Reply:
oh you are right, I forgot we have timelocks
but I still prefer to keep them dirty